### ISSUE TITLE
JSPM Internal Error

### ISSUE THREAD

**Issue by uwussimo** [Bestulo/deno-cheerio#5]
```
❯ deno cache xea.ts
Download https://dev.jspm.io/npm:lodash@4/flatten?dew
Download https://dev.jspm.io/npm:string_decoder@1?dew
error: Import 'https://dev.jspm.io/npm:string_decoder@1?dew' failed: 500 Internal Server Error
    at https://dev.jspm.io/npm:readable-stream@3.6.0/lib/_stream_readable.dew.js:11:50
```
I guess this is the end of jspm port's journey?

**Comment by traceypooh** [comment]
Seeing this too :( 
```
deno cache -r https://deno.land/x/cheerio@1.0.4/mod.ts
...
Download https://dev.jspm.io/npm:lodash@4.17.21/_baseMap.dew.js
Download https://dev.jspm.io/npm:lodash@4.17.21/map.dew.js
error: Import 'https://dev.jspm.io/npm:string_decoder@1?dew' failed: 500 Internal Server Error
    at https://dev.jspm.io/npm:readable-stream@3.6.0/lib/_stream_readable.dew.js:11:50
```

**Comment by Bestulo** [comment]
If you guys find a solution, please let me know. I'm currently very busy and can't do anything for a few days except maybe accepting a PR. There should be some way of importing the old versions. Anyway, I knew this was coming, it was too hacky to last.

**Comment by Bestulo** [comment]
I'm noticing that I can still get the jspm url. Wonder what the problem is.
I can debug in a couple of weeks maybe.

**Comment by uwussimo** [comment]
[This](https://github.com/Bestulo/deno-cheerio/pull/6#issue-1276076193) should probably help

**Comment by Bestulo** [comment]
Ty @uwussimo, let's hope your PR #6 solves this problem for everyone.

**Comment by traceypooh** [comment]
might be related to this -- I keep getting 500s for the url that pkgs are pegged to, best I can tell

https://dev.to/craigmorten/comment/10pa9

500s:
https://dev.jspm.io/npm:string_decoder@1?dew

works, best I can tell:
https://jspm.dev/npm:string_decoder@1?dew

**Comment by traceypooh** [comment]
so... how would I be able to use the merged fix?

I thought something like this should work, but I still get a 500:

```
deno cache -r https://raw.githubusercontent.com/Bestulo/deno-cheerio/main/mod.ts

Download https://raw.githubusercontent.com/Bestulo/deno-cheerio/main/mod.ts
Download https://raw.githubusercontent.com/Bestulo/deno-cheerio/main/types.ts
Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/index.dew.js
Download https://dev.jspm.io/npm:@jspm/core@1/nodelibs/buffer.js
Download https://dev.jspm.io/npm:cheerio-select@1?dew
Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/api/attributes.dew.js
Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/api/css.dew.js
Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/api/forms.dew.js
Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/api/manipulation.dew.js
Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/api/traversing.dew.js
Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/cheerio.dew.js
Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/index.dew.js
Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/load.dew.js
Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/options.dew.js
Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/parse.dew.js
Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/parsers/htmlparser2-adapter.dew.js
Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/parsers/parse5-adapter.dew.js
Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/static.dew.js
Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/types.dew.js
Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/utils.dew.js
Download https://dev.jspm.io/npm:dom-serializer@1?dew
Download https://dev.jspm.io/npm:domhandler@4?dew
Download https://dev.jspm.io/npm:htmlparser2@6?dew
Download https://dev.jspm.io/npm:parse5-htmlparser2-tree-adapter@6?dew
Download https://dev.jspm.io/npm:parse5@6?dew
Download https://dev.jspm.io/npm:tslib@2?dew
Download https://dev.jspm.io/npm:domelementtype@2?dew
Download https://dev.jspm.io/npm:domhandler@4.3.1/lib/index.dew.js
Download https://dev.jspm.io/npm:domhandler@4.3.1/lib/node.dew.js
Download https://dev.jspm.io/npm:@jspm/core@1.1.1/nodelibs/buffer.js
Download https://dev.jspm.io/npm:dom-serializer@1.4.1/lib/foreignNames.dew.js
Download https://dev.jspm.io/npm:dom-serializer@1.4.1/lib/index.dew.js
Download https://dev.jspm.io/npm:entities@2?dew
Download https://dev.jspm.io/npm:domelementtype@2.3.0/lib/index.dew.js
Download https://dev.jspm.io/npm:entities@2.2.0/lib/decode.dew.js
Download https://dev.jspm.io/npm:entities@2.2.0/lib/decode_codepoint.dew.js
Download https://dev.jspm.io/npm:entities@2.2.0/lib/encode.dew.js
Download https://dev.jspm.io/npm:entities@2.2.0/lib/index.dew.js
Download https://dev.jspm.io/npm:entities@2.2.0/lib/maps/decode.json.dew.js
Download https://dev.jspm.io/npm:entities@2.2.0/lib/maps/entities.json.dew.js
Download https://dev.jspm.io/npm:entities@2.2.0/lib/maps/legacy.json.dew.js
Download https://dev.jspm.io/npm:entities@2.2.0/lib/maps/xml.json.dew.js
error: Import 'https://dev.jspm.io/npm:htmlparser2@6?dew' failed: 500 Internal Server Error
    at https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/parse.dew.js:1:47
```

**Comment by uwussimo** [comment]
> so... how would I be able to use the merged fix?
> 
> I thought something like this should work, but I still get a 500:
> 
> ```
> demo cache -r https://raw.githubusercontent.com/Bestulo/deno-cheerio/main/mod.ts[](https://raw.githubusercontent.com/Bestulo/deno-cheerio/main/mod.ts)
> 
> Download https://raw.githubusercontent.com/Bestulo/deno-cheerio/main/mod.ts[](https://raw.githubusercontent.com/Bestulo/deno-cheerio/main/mod.ts)
> Download https://raw.githubusercontent.com/Bestulo/deno-cheerio/main/types.ts[](https://raw.githubusercontent.com/Bestulo/deno-cheerio/main/types.ts)
> Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/index.dew.js[](https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/index.dew.js)
> Download https://dev.jspm.io/npm:@jspm/core@1/nodelibs/buffer.js[](https://dev.jspm.io/npm:@jspm/core@1/nodelibs/buffer.js)
> Download https://dev.jspm.io/npm:cheerio-select@1?dew[](https://dev.jspm.io/npm:cheerio-select@1?dew)
> Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/api/attributes.dew.js[](https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/api/attributes.dew.js)
> Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/api/css.dew.js[](https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/api/css.dew.js)
> Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/api/forms.dew.js[](https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/api/forms.dew.js)
> Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/api/manipulation.dew.js[](https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/api/manipulation.dew.js)
> Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/api/traversing.dew.js[](https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/api/traversing.dew.js)
> Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/cheerio.dew.js[](https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/cheerio.dew.js)
> Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/index.dew.js[](https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/index.dew.js)
> Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/load.dew.js[](https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/load.dew.js)
> Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/options.dew.js[](https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/options.dew.js)
> Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/parse.dew.js[](https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/parse.dew.js)
> Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/parsers/htmlparser2-adapter.dew.js[](https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/parsers/htmlparser2-adapter.dew.js)
> Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/parsers/parse5-adapter.dew.js[](https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/parsers/parse5-adapter.dew.js)
> Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/static.dew.js[](https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/static.dew.js)
> Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/types.dew.js[](https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/types.dew.js)
> Download https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/utils.dew.js[](https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/utils.dew.js)
> Download https://dev.jspm.io/npm:dom-serializer@1?dew[](https://dev.jspm.io/npm:dom-serializer@1?dew)
> Download https://dev.jspm.io/npm:domhandler@4?dew[](https://dev.jspm.io/npm:domhandler@4?dew)
> Download https://dev.jspm.io/npm:htmlparser2@6?dew[](https://dev.jspm.io/npm:htmlparser2@6?dew)
> Download https://dev.jspm.io/npm:parse5-htmlparser2-tree-adapter@6?dew[](https://dev.jspm.io/npm:parse5-htmlparser2-tree-adapter@6?dew)
> Download https://dev.jspm.io/npm:parse5@6?dew[](https://dev.jspm.io/npm:parse5@6?dew)
> Download https://dev.jspm.io/npm:tslib@2?dew[](https://dev.jspm.io/npm:tslib@2?dew)
> Download https://dev.jspm.io/npm:domelementtype@2?dew[](https://dev.jspm.io/npm:domelementtype@2?dew)
> Download https://dev.jspm.io/npm:domhandler@4.3.1/lib/index.dew.js[](https://dev.jspm.io/npm:domhandler@4.3.1/lib/index.dew.js)
> Download https://dev.jspm.io/npm:domhandler@4.3.1/lib/node.dew.js[](https://dev.jspm.io/npm:domhandler@4.3.1/lib/node.dew.js)
> Download https://dev.jspm.io/npm:@jspm/core@1.1.1/nodelibs/buffer.js[](https://dev.jspm.io/npm:@jspm/core@1.1.1/nodelibs/buffer.js)
> Download https://dev.jspm.io/npm:dom-serializer@1.4.1/lib/foreignNames.dew.js[](https://dev.jspm.io/npm:dom-serializer@1.4.1/lib/foreignNames.dew.js)
> Download https://dev.jspm.io/npm:dom-serializer@1.4.1/lib/index.dew.js[](https://dev.jspm.io/npm:dom-serializer@1.4.1/lib/index.dew.js)
> Download https://dev.jspm.io/npm:entities@2?dew[](https://dev.jspm.io/npm:entities@2?dew)
> Download https://dev.jspm.io/npm:domelementtype@2.3.0/lib/index.dew.js[](https://dev.jspm.io/npm:domelementtype@2.3.0/lib/index.dew.js)
> Download https://dev.jspm.io/npm:entities@2.2.0/lib/decode.dew.js[](https://dev.jspm.io/npm:entities@2.2.0/lib/decode.dew.js)
> Download https://dev.jspm.io/npm:entities@2.2.0/lib/decode_codepoint.dew.js[](https://dev.jspm.io/npm:entities@2.2.0/lib/decode_codepoint.dew.js)
> Download https://dev.jspm.io/npm:entities@2.2.0/lib/encode.dew.js[](https://dev.jspm.io/npm:entities@2.2.0/lib/encode.dew.js)
> Download https://dev.jspm.io/npm:entities@2.2.0/lib/index.dew.js[](https://dev.jspm.io/npm:entities@2.2.0/lib/index.dew.js)
> Download https://dev.jspm.io/npm:entities@2.2.0/lib/maps/decode.json.dew.js[](https://dev.jspm.io/npm:entities@2.2.0/lib/maps/decode.json.dew.js)
> Download https://dev.jspm.io/npm:entities@2.2.0/lib/maps/entities.json.dew.js[](https://dev.jspm.io/npm:entities@2.2.0/lib/maps/entities.json.dew.js)
> Download https://dev.jspm.io/npm:entities@2.2.0/lib/maps/legacy.json.dew.js[](https://dev.jspm.io/npm:entities@2.2.0/lib/maps/legacy.json.dew.js)
> Download https://dev.jspm.io/npm:entities@2.2.0/lib/maps/xml.json.dew.js[](https://dev.jspm.io/npm:entities@2.2.0/lib/maps/xml.json.dew.js)
> error: Import 'https://dev.jspm.io/npm:htmlparser2@6?dew[](https://dev.jspm.io/npm:htmlparser2@6?dew)' failed: 500 Internal Server Error
>     at https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/parse.dew.js:1:47[](https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/lib/parse.dew.js:1:47)
> ```

seems like it's jspm dependent issue

**Comment by Bestulo** [comment]
Oh I forgot to make a new release haha. Lemme do that.

**Comment by Bestulo** [comment]
Nevermind I don't know how to do it from my phone. Will check later when I
can use my laptop.

**Comment by traceypooh** [comment]
thanks!  although, I think I'm pulling from your latest `main` branch commit from last 24h above (was going for bleeding edge to see if a `deno cache ..` would work)

**Comment by traceypooh** [comment]
I have no idea if this works, but FWIW this change to `mod.ts` in a checked out version of your repo at least `deno cache mod.ts` works <shrug>  :)


```
tracey@threepio  2:31AM  ~/dev/deno-cheerio
> dif mod.ts 
diff --git a/mod.ts b/mod.ts
index 200afbc..b1c7804 100644
--- a/mod.ts
+++ b/mod.ts
@@ -1,4 +1,4 @@
-import { dew } from "https://dev.jspm.io/npm:cheerio@1.0.0-rc.10/index.dew.js";
+import { dew } from "https://jspm.dev/npm:cheerio";
```

**Comment by traceypooh** [comment]
(Well, above change needed a bit more work, turns out -- had _runtime_ issues).

I *think* this is now working for me, best I can tell:

https://github.com/Bestulo/deno-cheerio/pull/7

**Comment by uwussimo** [comment]
@traceypooh good work comrade!

**Comment by Bestulo** [comment]
merged PRs and released 1.0.5, so it should now work from denoland import

thank you those who made this happen, chiefly @uwussimo and @traceypooh
